### PR TITLE
pages.es/*: add Spanish translation; npm: update page

### PR DESCRIPTION
**Pull request by Jeremy-Aza**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by Magrid0 — APPROVED** [review]
Looks good to me, thanks for your contribution!

File: pages.es/common/nest.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# nest
+
+> Herramienta de línea de comandos para inicializar, desarrollar y mantener aplicaciones Nest.
+> Mas información: <https://docs.nestjs.com/cli/overview>.
```
Comment (kant) [inline pages.es/common/nest.md:4]: Mas --> Más

File: pages.es/common/nest.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Mostrar la información sobre la versión de nest instalada:
```
Comment (kant) [inline pages.es/common/nest.md:6]: Mostrar --> Muestra

File: pages.es/common/nest.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`nest info`
+
+- Crear un nuevo proyecto NestJS en un directorio con el mismo nombre:
```
Comment (kant) [inline pages.es/common/nest.md:10]: Crear --> Crea

File: pages.es/common/nest.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`nest new {{nombre_del_proyecto}}`
+
+- Construir a proyecto NestJS específico:
```
Comment (kant) [inline pages.es/common/nest.md:14]: Construir --> Construye un

File: pages.es/common/nest.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`nest build {{nombre_del_proyecto}}`
+
+- Ejecutar un proyecto NestJS específico:
```
Comment (kant) [inline pages.es/common/nest.md:18]: Ejecutar --> Ejecuta

File: pages.es/common/nest.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`nest start {{nombre_del_proyecto}}`
+
+- Importar una librería al proyecto NestJS actual:
```
Comment (kant) [inline pages.es/common/nest.md:22]: Importar --> Importa

File: pages.es/common/netlify.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# netlify
+
+> Despliegue de sitios y configuración del despliegue continuo en la plataforma de Netlify.
+> Mas información: <https://cli.netlify.com>.
```
Comment (kant) [inline pages.es/common/netlify.md:4]: Mas --> Más

File: pages.es/common/netlify.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+- Acceder a la cuenta Netlify:
```
Comment (kant) [inline pages.es/common/netlify.md:6]: Acceder --> Accede

File: pages.es/common/netlify.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+`netlify login`
+
+- Desplegar el contenido de una directorio en Netlify:
```
Comment (kant) [inline pages.es/common/netlify.md:10]: Desplegar --> Despliega
una directorio --> un directorio

File: pages.es/common/netlify.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+`netlify deploy`
+
+- Configurar el despliegue continuo para un sitio nuevo o existente:
```
Comment (kant) [inline pages.es/common/netlify.md:14]: Configurar --> Configura

File: pages.es/common/netlify.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+`netlify init`
+
+- Iniciar un servidor local de desarrollo:
```
Comment (kant) [inline pages.es/common/netlify.md:18]: Iniciar --> Inicia

File: pages.es/common/next.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
+# next
+
+> Framework React que utiliza el renderizado del lado del servidor para crear aplicaciones web optimizadas.
+> Mas información: <https://nextjs.org/docs>.
```
Comment (kant) [inline pages.es/common/next.md:4]: Mas --> Más

File: pages.es/common/next.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+- Iniciar la aplicación actual en modo de desarrollo:
```
Comment (kant) [inline pages.es/common/next.md:6]: Iniciar --> Inicia

File: pages.es/common/next.md [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+
+`next dev`
+
+- Iniciar la aplicación actual y escuchar en un puerto específico:
```
Comment (kant) [inline pages.es/common/next.md:10]: Iniciar --> Inicia
escuchar --> escucha

File: pages.es/common/next.md [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+
+`next dev --port {{puerto}}`
+
+- Construir la aplicación actual optimizada para producción:
```
Comment (kant) [inline pages.es/common/next.md:14]: Construir --> Construye

File: pages.es/common/next.md [hunk 16]
```diff
@@ -0,0 +1,36 @@
...
+
+`next build`
+
+- Iniciar la aplicación compilada en modo de producción:
```
Comment (kant) [inline pages.es/common/next.md:18]: Iniciar --> Inicia

File: pages.es/common/next.md [hunk 17]
```diff
@@ -0,0 +1,36 @@
...
+
+`next start`
+
+- Iniciar la aplicación compilada y escuchar en un puerto específico:
```
Comment (kant) [inline pages.es/common/next.md:22]: Iniciar --> Inicia
escuchar --> escucha

File: pages.es/common/next.md [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+
+`next start --port {{puerto}}`
+
+- Exportar la aplicación actuala páginas HTML estáticas:
```
Comment (kant) [inline pages.es/common/next.md:26]: Exportar --> Exporta
actuala --> actual

File: pages.es/common/next.md [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+
+`next export`
+
+- Mostrar el estado de telemetría de Next.js:
```
Comment (kant) [inline pages.es/common/next.md:30]: Mostrar --> Muestra

File: pages.es/common/next.md [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+
+`next telemetry`
+
+- Mostrar la ayuda de un subcomando:
```
Comment (kant) [inline pages.es/common/next.md:34]: Mostrar --> Muestra

File: pages.es/common/nginx.md [hunk 21]
```diff
@@ -0,0 +1,24 @@
+# nginx
+
+> Servidor web Nginx.
+> Mas información: <https://nginx.org/en/>.
```
Comment (kant) [inline pages.es/common/nginx.md:4]: Mas --> Más

File: pages.es/common/nginx.md [hunk 22]
```diff
@@ -0,0 +1,24 @@
...
+
+- Iniciar el servidor con el archivo de configuración por defecto:
```
Comment (kant) [inline pages.es/common/nginx.md:6]: Iniciar --> Inicia

File: pages.es/common/nginx.md [hunk 23]
```diff
@@ -0,0 +1,24 @@
...
+
+`nginx`
+
+- Iniciar el servidor con un archivo de configuración personalizado:
```
Comment (kant) [inline pages.es/common/nginx.md:10]: Iniciar --> Inicia

File: pages.es/common/nginx.md [hunk 24]
```diff
@@ -0,0 +1,24 @@
...
+
+`nginx -c {{archivo_de_configuración}}`
+
+- Iniciar el servidor con un prefijo para todas las rutas relativas en el archivo de configuración:
```
Comment (kant) [inline pages.es/common/nginx.md:14]: Iniciar --> Inicia

File: pages.es/common/nginx.md [hunk 25]
```diff
@@ -0,0 +1,24 @@
...
+
+`nginx -c {{archivo_de_configuración}} -p {{prefijo/para/rutas/relativas}}`
+
+- Probar la configuración sin afectar al servidor en ejecución:
```
Comment (kant) [inline pages.es/common/nginx.md:18]: Probar --> Prueba

File: pages.es/common/npm.md [hunk 26]
```diff
@@ -0,0 +1,37 @@
+# npm
+
+> Gestor de paquetes JavaScript y Node.js.
+> Gestionar proyectos de Node.js y sus módulos de dependencias.
```
Comment (kant) [inline pages.es/common/npm.md:4]: Gestionar --> Gestiona

File: pages.es/common/vite.md [hunk 27]
```diff
@@ -0,0 +1,22 @@
+# Vite
+
+> Crear un proyecto Vite.
```
Comment (kant) [inline pages.es/common/vite.md:3]: Crear --> Crea

File: pages.es/common/vite.md [hunk 28]
```diff
@@ -0,0 +1,22 @@
...
+> Se utiliza para crear proyectos JavaScript.
+> Plantillas disponibles: vanilla, vanilla-ts, vue, vue-ts, react, react-ts, react-swc, react-swc-ts, preact, preact-ts, lit, lit-ts, svelte, svelte-ts.
+> Mas información: <https://vitejs.dev/guide>.
```
Comment (kant) [inline pages.es/common/vite.md:6]: Mas --> Más

File: pages.es/windows/choco.md [hunk 29]
```diff
@@ -0,0 +1,21 @@
+# choco
+
+> El gestor de paquetes Chocolatey.
+> Algunos subcomandos como `choco install` tienen su propia documentación de uso.
+> Mas información: <https://chocolatey.org>.
```
Comment (kant) [inline pages.es/windows/choco.md:5]: Mas --> Más

File: pages.es/windows/choco.md [hunk 30]
```diff
@@ -0,0 +1,21 @@
...
+
+- Ejecutar un comando Chocolatey:
```
Comment (kant) [inline pages.es/windows/choco.md:7]: Ejecutar --> Ejecuta

File: pages.es/windows/choco.md [hunk 31]
```diff
@@ -0,0 +1,21 @@
...
+
+`choco {{comando}}`
+
+- Llamar a la ayuda general:
```
Comment (kant) [inline pages.es/windows/choco.md:11]: Llamar --> Llama

File: pages.es/windows/choco.md [hunk 32]
```diff
@@ -0,0 +1,21 @@
...
+
+`choco -?`
+
+- Llamar a la ayuda sobre un comando específico:
```
Comment (kant) [inline pages.es/windows/choco.md:15]: Llamar --> Llama

File: pages.es/windows/choco.md [hunk 33]
```diff
@@ -0,0 +1,21 @@
...
+
+`choco {{comando}} -?`
+
+- Revisar la versión the Chocolatey:
```
Comment (kant) [inline pages.es/windows/choco.md:19]: Revisar --> Revisa
the --> de

File: pages.es/windows/cmd.md [hunk 34]
```diff
@@ -0,0 +1,36 @@
+# cmd
+
+> El intérprete de comandos de Windows.
+> Mas información: <https://learn.microsoft.com/windows-server/administration/windows-commands/cmd>.
```
Comment (kant) [inline pages.es/windows/cmd.md:4]: Mas --> Más

File: pages.es/windows/cmd.md [hunk 35]
```diff
@@ -0,0 +1,36 @@
...
+
+- Iniciar una sesión shell interactiva:
```
Comment (kant) [inline pages.es/windows/cmd.md:6]: Iniciar --> Inicia

File: pages.es/windows/cmd.md [hunk 36]
```diff
@@ -0,0 +1,36 @@
...
+
+`cmd`
+
+- Ejecutar [c]omandos específicos:
```
Comment (kant) [inline pages.es/windows/cmd.md:10]: Ejecutar --> Ejecuta

File: pages.es/windows/cmd.md [hunk 37]
```diff
@@ -0,0 +1,36 @@
...
+
+`cmd /c {{echo Hola Mundo}}`
+
+- Ejecutar un script específico:
```
Comment (kant) [inline pages.es/windows/cmd.md:14]: Ejecutar --> Ejecuta

File: pages.es/windows/cmd.md [hunk 38]
```diff
@@ -0,0 +1,36 @@
...
+
+`cmd {{ruta\al\script.bat}}`
+
+- Ejecutar comandos específicos y luego entrar en un shell interactivo:
```
Comment (kant) [inline pages.es/windows/cmd.md:18]: Ejecutar --> Ejecuta

File: pages.es/windows/cmd.md [hunk 39]
```diff
@@ -0,0 +1,36 @@
...
+
+`cmd /k {{echo Hola Mundo}}`
+
+- Iniciar una sesión shell interactiva donde `echo` está desactivado en la salida de comandos:
```
Comment (kant) [inline pages.es/windows/cmd.md:22]: Iniciar --> Inicia

File: pages.es/windows/cmd.md [hunk 40]
```diff
@@ -0,0 +1,36 @@
...
+
+`cmd /q`
+
+- Iniciar una sesión de shell interactiva con la expansión [v]ariable retardada activada o desactivada:
```
Comment (kant) [inline pages.es/windows/cmd.md:26]: Iniciar  -->Inicia

File: pages.es/windows/cmd.md [hunk 41]
```diff
@@ -0,0 +1,36 @@
...
+
+`cmd /v:{{on|off}}`
+
+- Iniciar una sesión shell interactiva con [e]xtensiones de comando activadas o desactivadas:
```
Comment (kant) [inline pages.es/windows/cmd.md:30]: Iniciar --> Inicia

File: pages.es/windows/del.md [hunk 42]
```diff
@@ -0,0 +1,37 @@
+# del
+
+> Elimina uno o más archivos.
+> En PowerShell, este comando es un alias de `Remove-Item`. Esta documentación se basa en la versión del símbolo del sistema (`cmd`) de `del`.
+> Mas información: <https://learn.microsoft.com/windows-server/administration/windows-commands/del>.
```
Comment (kant) [inline pages.es/windows/del.md:5]: Mas --> Más

File: pages.es/windows/del.md [hunk 43]
```diff
@@ -0,0 +1,37 @@
...
+
+- Elimina uno o más archivos o patrones separados por espacios:
+
+`del {{patrón_del_archivo}}`
+
+- Solicita confirmación antes de borrar cada archivo:
+
+`del {{patrón_del_archivo}} /p`
+
+- Forzar la eliminación de archivos de sólo lectura:
```
Comment (kant) [inline pages.es/windows/del.md:19]: Forzar --> Fuerza

File: pages.es/windows/del.md [hunk 44]
```diff
@@ -0,0 +1,37 @@
...
+
+- Eliminar de forma recursiva archivos de todos los subdirectorios:
+
+`del {{patrón_del_archivo}} /s`
+
+- No generar una solicitud de confirmación al eliminar archivos basados en un comodín global:
+
+`del {{patrón_del_archivo}} /q`
+
+- Mostrar la ayuda y la lista de atributos disponibles:
```
Comment (kant) [inline pages.es/windows/del.md:31]: Mostrar --> Muestra

File: pages.es/windows/dir.md [hunk 45]
```diff
@@ -0,0 +1,24 @@
+# dir
+
+> Listar el contenido del directorio.
```
Comment (kant) [inline pages.es/windows/dir.md:3]: Listar --> Lista

File: pages.es/windows/dir.md [hunk 46]
```diff
@@ -0,0 +1,24 @@
...
+> Mas información: <https://learn.microsoft.com/windows-server/administration/windows-commands/dir>.
```
Comment (kant) [inline pages.es/windows/dir.md:4]: Mas --> Más

File: pages.es/windows/dir.md [hunk 47]
```diff
@@ -0,0 +1,24 @@
...
+
+- Mostrar el contenido del directorio actual:
+
+`dir`
+
+- Mostrar el contenido de un directorio determinado:
```
Comment (kant) [inline pages.es/windows/dir.md:10]: Mostrar --> Muestra

File: pages.es/windows/dir.md [hunk 48]
```diff
@@ -0,0 +1,24 @@
...
+
+`dir {{ruta\al\directorio}}`
+
+- Mostrar el contenido del directorio actual, incluidos los ocultos:
```
Comment (kant) [inline pages.es/windows/dir.md:14]: Mostrar --> Muestra

File: pages.es/windows/dir.md [hunk 49]
```diff
@@ -0,0 +1,24 @@
...
+
+`dir /a`
+
+- Mostrar el contenido de un directorio determinado, incluidos los ocultos:
```
Comment (kant) [inline pages.es/windows/dir.md:18]: Mostrar --> Muestra

File: pages.es/windows/dir.md [hunk 50]
```diff
@@ -0,0 +1,24 @@
...
+
+`dir {{ruta\al\directorio}} /a`
+
+- Mostrar una lista de directorios y archivos, sin información adicional:
```
Comment (kant) [inline pages.es/windows/dir.md:22]: Mostrar  --> Muestra

File: pages.es/windows/mkdir.md [hunk 51]
```diff
@@ -0,0 +1,12 @@
+# mkdir
+
+> Crea un directorio.
+> Mas información: <https://learn.microsoft.com/windows-server/administration/windows-commands/mkdir>.
```
Comment (kant) [inline pages.es/windows/mkdir.md:4]: Mas --> Más

File: pages.es/windows/mkdir.md [hunk 52]
```diff
@@ -0,0 +1,12 @@
...
+
+- Crea un directorio:
+
+`mkdir {{ruta\al\directorio}}`
+
+- Crear un árbol de directorios anidado de forma recursiva:
```
Comment (kant) [inline pages.es/windows/mkdir.md:10]: Crear --> Crea

File: pages.es/windows/nvm.md [hunk 53]
```diff
@@ -0,0 +1,25 @@
+# nvm
+
+> Instalar, desinstalar o cambiar entre versiones de Node.js.
```
Comment (kant) [inline pages.es/windows/nvm.md:3]: Instalar --> Instala
desinstalar --> desinstala

File: pages.es/windows/nvm.md [hunk 54]
```diff
@@ -0,0 +1,25 @@
...
+> Admite números de versión como "12.8" o "v16.13.1", y etiquetas como "stable", "system", etc.
+> Mas información: <https://github.com/coreybutler/nvm-windows>.
```
Comment (kant) [inline pages.es/windows/nvm.md:5]: Mas --> Más

File: pages.es/windows/nvm.md [hunk 55]
```diff
@@ -0,0 +1,25 @@
...
+
+- Instala una versión específica de Node.js:
+
+`nvm install {{versión_de_node}}`
+
+- Establecer la versión por defecto de Node.js (debe ejecutarse como Administrador):
```
Comment (kant) [inline pages.es/windows/nvm.md:11]: Establecer --> Establece

**Review by kant — CHANGES_REQUESTED** [review]

**Comment by kbdharun** [comment]
Btw, Kant you can make easily committable/applicable code suggestions (for a whole page) on GitHub instead of manually typing the code change.

When reviewing the page you can click on this icon to add a suggestion:

image

Feel free to checkout https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request

File: pages.es/common/helm-install.md [hunk 56]
```diff
@@ -0,0 +1,32 @@
...
+
+`helm install {{nombre}} {{nombre_del_repositorio}}/{{nombre_del_chart}} --dry-run`
+
+- Instalar un chart de helm con valores personalizados:
+
+`helm install {{nombre}} {{nombre_del_repositorio}}/{{nombre_del_chart}} --set {{parámetro1}}={{valor1}},{{parámetr2}}={{valor2}}`
+
+- Instalar un chart de helm pasando un archivo de valores personalizados:
+
+`helm install {{nombre}} {{nombre_del_repositorio}}/{{nombre_del_chart}} --values {{ruta/a/valores.yaml}}`
```
Comment (kbdharun) [inline pages.es/common/helm-install.md:32]:
```suggestion
> Instalar un chart de helm.
> Más información: <https://helm.sh/docs/intro/using_helm/#helm-install-installing-a-package>.

- Instala un chart de helm:

`helm install {{nombre}} {{nombre_del_repositorio}}/{{nombre_del_chart}}`

- Instala un chart de helm desde un directorio de chart desempaquetado:

`helm install {{nombre}} {{ruta/al/directorio_de_origen}}`

- Instala un chart de helm desde una URL:

`helm install {{nombre_del_paquete}} {{https://example.com/charts/packagename-1.2.3.tgz}}`

- Instalar un chart de helm y genera un nombre:

`helm install {{nombre_del_repositorio}}/{{nombre_del_chart}} --generate-name`

- Realiza una simulación:

`helm install {{nombre}} {{nombre_del_repositorio}}/{{nombre_del_chart}} --dry-run`

- Instala un chart de helm con valores personalizados:

`helm install {{nombre}} {{nombre_del_repositorio}}/{{nombre_del_chart}} --set {{parámetro1}}={{valor1}},{{parámetro2}}={{valor2}}`

- Instala un chart de helm pasando un archivo de valores personalizados:

`helm install {{nombre}} {{nombre_del_repositorio}}/{{nombre_del_chart}} --values {{ruta/a/valores.yaml}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for this massive PR ❤️ . (I have manually applied all of @kant's suggestions)

**Review by kant — APPROVED** [review]

**Pull request merged**